frontend/modules/issues/issues.sagas.ts

```diff
@@ -154,10 +153,7 @@ export function* saveIssue({ teamspace, model, issueData, revision }) {
 
 		const { data: savedIssue } = yield API.saveIssue(teamspace, model, issue);
 
-		AnalyticService.sendEvent({
-			eventCategory: 'Issue',
-			eventAction: 'create'
-		});
+		analyticsService.sendEvent(EVENT_CATEGORIES.issue, EVENT_ACTIONS.create );
```

Comment:
You changed the case of constant name, but you forgot about constant's field, e.g
`EVENT_CATEGORIES.issue => EVENT_CATEGORIES.ISSUE`